src/vue/pages/Funds.vue

```diff
@@ -1,61 +1,403 @@
 <template>
   <div class="funds">
     <top-bar>
+      <template slot="main">
+        <select-field
+          :disabled="!isLoaded"
+          v-model="filters.baseAsset"
+          :values="fundAssets"
+          class="app__select app__select--no-border"
+        />
+
+        <select-field
+          :disabled="!isLoaded"
+          v-model="filters.state"
+          :values="FUND_STATES"
+          class="app__select app__select--no-border"
+        />
+
+        <select-field
```

Comment:
let's remove the sorting feature, because new API won't temporary support it

Reply:
Removed